caikit/core/__init__.py

```diff
@@ -42,3 +42,5 @@
 extract = MODEL_MANAGER.extract
 load = MODEL_MANAGER.load
 resolve_and_load = MODEL_MANAGER.resolve_and_load
+train = MODEL_MANAGER.train
+get_model_future = MODEL_MANAGER.get_model_future
```

Comment:
do we need to expose this at this level? This seems like more of an internal use only?

Reply:
I think we do want to expose this here (and in the top of `caikit`) since they would be how, for example, you'd launch a remote training job from a notebook with a non-local trainer